src/Extension/PhiremockProcess.php

```diff
@@ -52,32 +52,20 @@ public function start($ip, $port, $path, $logsPath, $debug, $expectationsPath)
         $phiremockPath = is_file($path) ? $path : $path . DIRECTORY_SEPARATOR . 'phiremock';
         $expectationsPath = is_dir($expectationsPath) ? $expectationsPath : '';
 
-        $this->logPhiremockCommand($ip, $port, $debug, $expectationsPath, $phiremockPath);
         $this->initProcess($ip, $port, $debug, $expectationsPath, $phiremockPath);
+        $this->logPhiremockCommand($debug);
         $logFile = $logsPath . DIRECTORY_SEPARATOR . self::LOG_FILE_NAME;
         $this->process->start(function ($type, $buffer) use ($logFile) {
             file_put_contents($logFile, $buffer, FILE_APPEND);
         });
-        $this->setUpProcessCompatibility();
     }
 
     /**
      * Stops the process.
      */
     public function stop()
     {
-        if ($this->isPcntlEnabled()) {
-            $this->process->signal(SIGTERM);
-            $this->process->stop(3, SIGKILL);
-        }
-    }
-
-    private function setUpProcessCompatibility()
-    {
-        $this->process->setEnhanceSigchildCompatibility(true);
-        if ($this->isWindows()) {
-            $this->process->setEnhanceWindowsCompatibility(true);
```

Comment:
The `setEnhanceSigchildCompatibility` method is deprecated since version 3.3 and will be removed in 4.0. Sigchild compatibility will always be enabled.
The `setEnhanceWindowsCompatibility` method is deprecated since version 3.3 and will be removed in 4.0. Enhanced Windows compatibility will always be enabled.